### DISCUSSION TITLE
Manual mesh and z offset work together?

### DISCUSSION THREAD

**Discussion by Tapap84** [Jyers/Marlin#1045] [General Discussion]
Howdy. I got my mesh set and haven't had to tweak it in over 3mo. I just installed a new nozzle and it's smaller. A z adjustment of -0.04 is where I need to be.

If I set a z offset of -0.04, will the printer respect both the z offset and the mesh?

Would it be better to run a new mesh and just bring each point 0.04 closer to the bed?

**Comment by Jyers** [comment]
A little z offset is just what you need, there should be no problem applying that even with the mesh :)
